simple_ddl_parser/utils.py

```diff
@@ -1,12 +1,31 @@
 import re
-from typing import List
+from typing import List, Tuple, Optional, Union, Any
 
+__all__ = [
+    "remove_par",
+    "check_spec",
+    "find_first_unpair_closed_par",
+    "normalize_name",
+    "get_table_id",
+]
 
-def remove_par(p_list: List[str]) -> List[str]:
-    remove_list = ["(", ")"]
-    for symbol in remove_list:
-        while symbol in p_list:
-            p_list.remove(symbol)
+_parentheses = ('(', ')')
+
+
+def remove_par(p_list: List[Union[str, Any]]) -> List[Union[str, Any]]:
+    """
+    Remove the parentheses from the given list
+
+    Warn: p_list may contain unhashable types for some unexplored reasons
```

Comment:
It's better to find the reason why `p_list` may contain `dict`. Is it expected?

Reply:
yes, it is expected, p_list contains results of parsing statements, usually it something like {'column': name, 'unique': True}, so p_list - always list, but elements of this list can be dicts

Reply:
Ok, I rewrote this docstring then.